docs/testrunner-toolkit/configuration.md

```diff
@@ -159,7 +160,7 @@ If you wish to execute tests on different screen resolutions while using Testrun
 
 __Example__:
 
-```yaml
+```yaml {19-21}
```

Comment:
Thanks for catching that. It was an older update I made to simply highlight the lines, but I think you are right -- it would be better to just pull from the example repo. Will fix that.

Reply:
Happy to add it to the example repo :)

Reply:
Added: https://github.com/saucelabs/saucectl-cypress-example/blob/master/.sauce/config.yml#L25

Keep in mind that references like these can easy break if the config changes in the referenced repo (e.g. if reference lines are re-ordered).

So it's better to not simply link to the master branch, but rather reference by commit:
https://github.com/saucelabs/saucectl-cypress-example/blob/4141ba08e501ae2be691969a1626c78c18a77aff/.sauce/config.yml#L25